From 8c29731192565b9c917d6b97db78dcd302283df8 Mon Sep 17 00:00:00 2001
From: Ulrich Drepper <drepper@gmail.com>
Date: Tue, 31 May 2011 14:23:01 -0400
Subject: [PATCH] Fix typo in  stack guard setup code for old kernels

---
 ChangeLog                           |    5 +++++
 sysdeps/unix/sysv/linux/dl-osinfo.h |    2 +-
 2 files changed, 6 insertions(+), 1 deletions(-)

2011-05-31  Ulrich Drepper  <drepper@gmail.com>

	* sysdeps/unix/sysv/linux/dl-osinfo.h (_dl_setup_stack_chk_guard): Fix
	typo.

diff --git a/sysdeps/unix/sysv/linux/dl-osinfo.h b/sysdeps/unix/sysv/linux/dl-osinfo.h
index eb7fedc..28fce4f 100644
--- a/sysdeps/unix/sysv/linux/dl-osinfo.h
+++ b/sysdeps/unix/sysv/linux/dl-osinfo.h
@@ -81,7 +81,7 @@ _dl_setup_stack_chk_guard (void *dl_random)
 	{
 	  ssize_t reslen = read_not_cancel (fd, ret.bytes + 1, filllen);
 	  close_not_cancel_no_status (fd);
-	  if (reslen == (ssize_) filllen)
+	  if (reslen == (ssize_t) filllen)
 	    return ret.num;
 	}
 # endif
-- 
1.7.6

